packages/svelteui-core/src/lib/components/Code/CopyIcon.svelte

```diff
@@ -10,12 +10,12 @@
 			fill-rule="evenodd"
 			clip-rule="evenodd"
 		/>
+	{:else}
```

Comment:
The else isn't needed here but that's okay

Reply:
I added because both the clipboard and the check were appearing when checked